diff --git a/node_modules/@flyerhq/react-native-link-preview/lib/LinkPreview.js b/node_modules/@flyerhq/react-native-link-preview/lib/LinkPreview.js
index 7ab00a6..d9e3a4f 100644
--- a/node_modules/@flyerhq/react-native-link-preview/lib/LinkPreview.js
+++ b/node_modules/@flyerhq/react-native-link-preview/lib/LinkPreview.js
@@ -77,7 +77,7 @@ export const LinkPreview = React.memo(({ containerStyle, enableAnimation, header
                     textContainerStyle,
                 ]) },
                 renderHeaderNode(),
-                renderTextNode(),
+             //   renderTextNode(),
                 ((data === null || data === void 0 ? void 0 : data.description) ||
                     ((data === null || data === void 0 ? void 0 : data.image) &&
                         aspectRatio === 1 &&
@@ -116,9 +116,11 @@ export const LinkPreview = React.memo(({ containerStyle, enableAnimation, header
 const styles = StyleSheet.create({
     description: {
         marginTop: 4,
+        color:"black"
     },
     header: {
         marginBottom: 6,
+        color:"black"
     },
     image: {
         alignSelf: 'center',
@@ -126,7 +128,7 @@ const styles = StyleSheet.create({
     },
     metadataContainer: {
         flexDirection: 'row',
-        marginTop: 16,
+        marginTop: -2,
     },
     metadataTextContainer: {
         flex: 1,
@@ -138,11 +140,12 @@ const styles = StyleSheet.create({
         width: 48,
     },
     textContainer: {
-        marginHorizontal: 24,
+        marginHorizontal: 10,
         marginVertical: 16,
     },
     title: {
         fontWeight: 'bold',
+        color:"black"
     },
 });
 //# sourceMappingURL=LinkPreview.js.map
\ No newline at end of file
