diff --git a/node_modules/@flyerhq/react-native-link-preview/lib/LinkPreview.js b/node_modules/@flyerhq/react-native-link-preview/lib/LinkPreview.js
index 7ab00a6..59237c4 100644
--- a/node_modules/@flyerhq/react-native-link-preview/lib/LinkPreview.js
+++ b/node_modules/@flyerhq/react-native-link-preview/lib/LinkPreview.js
@@ -77,7 +77,7 @@ export const LinkPreview = React.memo(({ containerStyle, enableAnimation, header
                     textContainerStyle,
                 ]) },
                 renderHeaderNode(),
-                renderTextNode(),
+             //   renderTextNode(),
                 ((data === null || data === void 0 ? void 0 : data.description) ||
                     ((data === null || data === void 0 ? void 0 : data.image) &&
                         aspectRatio === 1 &&
@@ -126,7 +126,7 @@ const styles = StyleSheet.create({
     },
     metadataContainer: {
         flexDirection: 'row',
-        marginTop: 16,
+        marginTop: -5,
     },
     metadataTextContainer: {
         flex: 1,
@@ -138,7 +138,7 @@ const styles = StyleSheet.create({
         width: 48,
     },
     textContainer: {
-        marginHorizontal: 24,
+        marginHorizontal: 10,
         marginVertical: 16,
     },
     title: {
